spring-cloud-azure-messaging/src/main/java/com/microsoft/azure/spring/messaging/container/AbstractAzureListenerContainerFactory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.messaging.container;
+
+import com.microsoft.azure.spring.integration.core.api.SubscribeByGroupOperation;
+import com.microsoft.azure.spring.messaging.endpoint.AbstractAzureListenerEndpoint;
+import com.microsoft.azure.spring.messaging.endpoint.AzureListenerEndpoint;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
+/**
+ * Base {@link ListenerContainerFactory} for Spring's base container implementation.
+ *
+ * @param <C> the container type
+ * @author Warren Zhu
+ * @see AbstractAzureListenerEndpoint
+ */
+
```

Comment:
Let's unify spacing in all files.

Reply:
I suppose IDE should have this kind of format. Let me see what's wrong with this.